rohit/Image-Search-Hooks/src/component/ImageCard.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+class ImageCard extends React.Component {
```

Comment:
Let's convert this into functional as well? Why waste opportunity to practice some more!